### ISSUE TITLE
jq doesn't work

### ISSUE THREAD

**Issue by m-titov** [paolosimone/virtual-json-viewer#22]
```
 curl https://wttr.in/Detroit\?format\=j1 | jq '.weather|.[].date'                                  [2022-05-19 23:58:52]
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 48898  100 48898    0     0   112k      0 --:--:-- --:--:-- --:--:--  115k
"2022-05-19"
"2022-05-20"
"2022-05-21"
```
works as expected but Virtual Json Viewer plugin throws `Unexpected string in JSON at position 13` error for the same expression:
`.weather|.[].date` 

See also [https://jqplay.org/s/DHmRZozMOj9](https://jqplay.org/s/DHmRZozMOj9)
Could you please fix it and make my life easier?

**Comment by paolosimone** [comment]
Hi @m-titov, first of all thanks for using my plugin!

**TLDR;**

use `.weather | map(.date)` instead

---

**Complete explanation**

Virtual Json Viewer is a plugin to render json in the browser, so jq commands must output a valid json.

Unfortunately the following is not a valid json array
```text
"2022-05-19"
"2022-05-20"
"2022-05-21"
```

While this is
```bash
curl -s https://wttr.in/Detroit\?format\=j1 | jq '.weather | map(.date)'
[
  "2022-05-20",
  "2022-05-21",
  "2022-05-22"
]
```

**Comment by paolosimone** [comment]
Since this behaviour is a bit tricky I added a more in-depth section to the readme

https://github.com/paolosimone/virtual-json-viewer#notes-on-jq
